scripts/timelapse-colorizer-data/data_writer_utils.py

```diff
@@ -161,6 +161,7 @@ def write_manifest(
         self,
         num_frames: int,
         feature_names: List[str],
+        feature_metadata: List[object] = [],
```

Comment:
At least, this should have stricter typing (enforcing `unit`) and at best, maybe the api should just accept a list of feature units as List[str] instead of asking users to structure it into some not-very-well-documented metadata object.   The utils code could take care of the structuring.

Reply:
Keeping this as a dictionary because I think this will keep things a bit tidier if the API updates in the future?

Reply:
It's a small point for now, but my take is: this is tidier for devs but not necessarily any easier for data entry.  my goal is the lowest possible friction for a scientist to throw their own script together.